exporter/collector/metrics.go

```diff
@@ -455,13 +472,64 @@ func (me *MetricsExporter) export(ctx context.Context, req *monitoringpb.CreateT
 	}
 
 	// always record the number of successful points
-	recordPointCountDataPoint(ctx, succeededPoints, "OK")
+	me.pointsExportedCounter.Add(
+		ctx,
+		int64(succeededPoints),
+		metricapi.WithAttributes(attribute.String("status", "OK")),
+	)
 	if failedPoints > 0 {
-		recordPointCountDataPoint(ctx, failedPoints, st)
+		st := statusCodeToString(s)
+		me.pointsExportedCounter.Add(
+			ctx,
+			int64(failedPoints),
+			metricapi.WithAttributes(attribute.String("status", st)),
+		)
 	}
 	return err
 }
 
+func statusCodeToString(s *status.Status) string {
```

Comment:
could you use [`status.Code().String()`](https://github.com/grpc/grpc-go/blob/2da976983bbb33feb3e25b7daaa8f60b9769adb5/codes/code_string.go#L31) instead of redeclaring the function here?

Reply:
That would change the codes from e.g. `PERMISSION_DENIED` to `PermissionDenied`.  The capitalized version appears to be what the gRPC instrumentation will give us, so I suspect it was done to make the labels the same between this metric and the grpc metric.  I would prefer to leave it as-is for this PR so we don't make any breaking changes here.

Reply:
feel free to open a follow-up issue if you think it is worth doing.